src/tests/testBoilerplate.cc

```diff
@@ -7,11 +7,32 @@
  */
 
 #include "squid.h"
-#include "testBoilerplate.h"
+#include "compat/cppunit.h"
 #include "unitTestMain.h"
 
 #include <stdexcept>
 
+/*
+ * demonstration test file, as new idioms are made they will
+ * be shown in the TestBoilerplate source.
+ *
+ * If a class declaration is only used in more than one source file,
+ * then move the declaration to a separate header file
```

Comment:
"only used in more than one" phrasing sounds odd to me.

If you insist on having this text:

```suggestion
 *
 * If the class declaration is used in more than one source file,
 * then place that declaration into a dedicated header file instead.
```

But my recommendation is to remove it. There is no (reasonable) way to reuse a class declaration across source files without placing it into a header file. We do not need to document this:

```suggestion
```